libnmstate/nmclient.py

```diff
@@ -39,3 +39,16 @@ def client(refresh=False):
     if _nmclient is None or refresh:
         _nmclient = NM.Client.new(None)
     return _nmclient
+
+
+def connection_ensure_setting(connection, setting_type):
```

Comment:
This is used in the next patch?

Reply:
Sorry, this was left from before and can go.